django_social_share/templates/django_social_share/templatetags/chama_no_zap_bb.html

```diff
@@ -0,0 +1,3 @@
+<div class="zap-this">
```

Comment:
can we call this template just `whatsapp.html`?

Reply:
Yes.